spec/bson/logstash_timestamp_spec.rb

```diff
@@ -0,0 +1,30 @@
+# encoding: utf-8
+require_relative "../spec_helper"
+require 'stringio'
+
+describe ::LogStash::Timestamp do
```

Comment:
I see a pattern for this spec, can we might be make a shared example with the other bson_type specs? what do you think?